src/utils/itemValidator.js

```diff
@@ -0,0 +1,27 @@
+export const isInputEmpty = (userInput) => {
+	const trimmedUserInput = userInput.trim();
+	return trimmedUserInput ? false : true;
```

Comment:
hello! This is a nitpick; I am wondering why you used a ternary operator here instead of just using the ! operator. for example, would in line 3 using `return !trimmedUserInput` not achieve the same result?

Reply:
We had written this out 3 ways, `return !trimmedUserInput` will return the same result and can further refactor it.